backend/cmd/headlamp.go

```diff
@@ -1384,3 +1388,122 @@ func absPath(path string) (string, error) {
 
 	return filepath.Join(currentUser.HomeDir, path[2:]), nil
 }
+
+//nolint:funlen
+func (c *HeadlampConfig) handleNodeDrain(w http.ResponseWriter, r *http.Request) {
+	var drainPayload struct {
+		Cluster  string `json:"cluster"`
+		NodeName string `json:"nodeName"`
+	}
+	// get nodeName and cluster from request body
+	err := json.NewDecoder(r.Body).Decode(&drainPayload)
+	if err != nil {
+		http.Error(w, "Error decoding payload", http.StatusBadRequest)
+		return
+	}
```

Comment:
Can you please remove `error` keyword from the error message. For context please refer https://github.com/headlamp-k8s/headlamp/pull/1129#discussion_r1193068710

```suggestion
        if err := json.NewDecoder(r.Body).Decode(&drainPayload); err != nil {
		http.Error(w, "Decoding payload", http.StatusBadRequest)
		return
	}
```

Reply:
@knrt10 thanks for looking into this, I looked at the comment and it makes sense to not mention the error with logs coz they already explicitly specify if it's an error, but in this case, the server sends a 400 status code and it sends this error string as a response so for a client I think it makes sense to have the error word here so looking at the response string it is explicit  that It was an error while decoding payload,